packages/ag-charts-enterprise/src/features/error-bar/errorBarNode.ts

```diff
@@ -170,52 +200,57 @@ export class ErrorBarNode extends _Scene.Group {
         }
         caps.path.closePath();
         caps.markDirtyTransform();
+    }
+
+    updateBBoxes(...ranges: InteractionRange[]): void {
```

Comment:
I'm just wondering if we should improve the scene-graph support for this type of operation, rather than have all of this behaviour as `ErrorBar`-specific functionality?

`Group.computeBBox()` already exists, and I think we could improve the `Group.containsPoint()` handling for this case (specifically, allow `Group` to configurably have accurate bounding-box handling)?

Reply:
I agree that this should be a built-in scene graph feature. However, adding such a feature to the scene has the potential of causing many bug and performance regressions across the entire ag-charts framework.

Are we confident enough with our test coverage to mitigate this risk so close the initial release? Or should we leave this in the backlog?

Reply:
Some this has been moved into the community package (new file _nearest.ts_), but the `ErrorBarNode` still cares take of calculating the size & position of bounding boxes. PTAL.